### PR TITLE
Update help prompts to fit new name + add formerly: line for old script name.

### PR DESCRIPTION
**Pull request by gagnonanthony**

### CODE REVIEW

**Comment by CHrlS98** [comment]
I think the word you are looking for is "formerly" and not "formally"!

image

**Comment by gagnonanthony** [comment]
Yep that's true, my bad @CHrlS98

**Comment by AntoineTheb** [comment]
image

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scripts/scil_bundle_pairwise_comparison.py [hunk 1]
```diff
@@ -94,12 +96,14 @@ def load_data_tmp_saving(args):
     disable_centroids = args[3]
 
     # Since data is often re-use when comparing multiple bundles, anything
```
Comment (AntoineTheb) [inline scripts/scil_bundle_pairwise_comparison.py:98]: re-used

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**